packages/explorer/src/consts.ts

```diff
@@ -0,0 +1,38 @@
+import { Hex } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+import { chainState, mudStoreTables } from "@latticexyz/store-sync/sqlite";
+
+type Table = {
+  [key: symbol]: string;
+};
+
+const TableName = Symbol.for("drizzle:Name");
+export const NON_EDITABLE_TABLES = [
+  (chainState as unknown as Table)[TableName],
+  (mudStoreTables as unknown as Table)[TableName],
+];
```

Comment:
I think you can use `getTableName` from drizzle to do this

```ts
import { getTableName } from "drizzle-orm";
const internalTableNames = [chainState, mudStoreTables].map(getTableName);
```

but if we're not using drizzle here, it probably doesn't make much sense to add it just for this, so maybe we export these table names from store-sync/sqlite?

Reply:
Yeah, tried to do that but as you said, not great to import the whole package for one function. And then it still didn't work because of "barrel files imports" issue due to some incompatibility between Next.js and the package itself. So sure, exporting from `store-sync/sqlite` makes most sense to me, will create a PR.